interface/usergroup/facility_admin.php

```diff
@@ -232,7 +232,7 @@ function displayAlert()
                     if ($facility['pos_code'] == $pos['code']) {
                         echo "selected";
                     }
-                    echo ">" . $pos['code']  . ": ". $pos['title'];
+                    echo ">" . $pos['code']  . ": ". xlt($pos['title']);
```

Comment:
Makes much more sense to wrap xl() around the titles in the original array and then just use text() here.
By wrapping the original titles will then mean it will get picked up by the translation pipeline scripts(and will end up in the google docs spreadsheet at some point).